### PR TITLE
Zane

### PR DESCRIPTION
**Pull request by DogsAreTheShit**
changed from cards to lists. added a visit button which can, later on, be used to redirect the user to a document's individual page. Let me know of any changes I can make

### CODE REVIEW

**Comment by DogsAreTheShit** [comment]
I committed a change, don't know if this updates the pull request tho. I added spacers and added elevation for only hovering

File: pages/search.vue [hunk 1]
```diff
@@ -1,32 +1,58 @@
...
-      <v-card color="primary" dark>
-        <div class="d-flex flex-no-wrap justify-space-between">
-          <div>
-            <v-card-title>{{ doc._source.meta.title }}</v-card-title>
-            <v-card-subtitle>{{ doc._source.file.filename }}</v-card-subtitle>
-            <v-card-subtitle>{{ doc._source.file.url }}</v-card-subtitle>
-            <v-card-subtitle>
+    <!-- Used md="9" for 9-column width of the v-col -->
+    <v-col v-for="(doc, i) in hits" :key="i + '-' + doc._id" cols="12" md="9">
+      <v-card max-width="1000" class="mx-auto" hover>
```
Comment (anewage) [inline pages/search.vue:5]: Adding "outlined" prop would also help in enhancing the UX of cards.

File: pages/search.vue [hunk 2]
```diff
@@ -1,32 +1,58 @@
...
+            </v-list-item-content>
+          </v-list-item>
+        </v-list>
       </v-card>
+      <v-divider></v-divider>
+      <v-divider></v-divider>
+      <v-divider></v-divider>
+      <v-divider></v-divider>
+      <v-divider></v-divider>
+      <v-divider></v-divider>
```
Comment (anewage) [inline pages/search.vue:48]: I think these lines are due to an error. We only need one divider (if any).

Comment (DogsAreTheShit) [reply pages/search.vue:48]: I repeated the code to make the divider larger. If there is a simpler way of going about it I'll implement that, or if a small divider works too I can take out those extra lines

**Comment by DogsAreTheShit** [comment]
Added outlined to the cards, removed excessive divider code. Let me know if there is anything else I need to change before my branch can be merged

**Pull request merged**